ext/src/Duration.c

```diff
@@ -7,36 +7,69 @@
 
 #include "Duration.h"
 
+#define CASS_INT32_MAX 2147483647L
```

Comment:
I think we should be consistent with the other uses of MAX/MIN in our code base and handle the cases where it is not defined by a particular platform (see [Bigint](https://github.com/datastax/php-driver/blob/e11b36df381d08dd321cdc22e33c7724fe6d6f07/ext/src/Bigint.c#L23-L26))

```
#if !defined(HAVE_STDINT_H) && !defined(_MSC_STDINT_H_)
#  define INT32_MAX 2147483647L
#  define INT32_MIN (-INT32_MAX-1)
#endif
```